renaissance/scripts/perf/parsemetrics-promql.sh

```diff
@@ -0,0 +1,395 @@
+#!/bin/bash
+#
+# Copyright (c) 2020, 2021,2022 IBM Corporation, RedHat and others.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+### Script to parse prometheus query data###
+
+
+CURRENT_DIR="$(dirname "$(realpath "$0")")"
+source ${CURRENT_DIR}/../utils/common.sh
+
+# Parse CPU, memory and cluster information
+# input:type of run(warmup|measure), total number of runs, iteration number
+# output:Creates cpu, memory and cluster information in the form of log files for each run
+function parsePromMetrics()  {
+	TYPE=$1
+	TOTAL_RUNS=$2
+	ITR=$3
+
+	for (( run=0 ; run<"${TOTAL_RUNS}" ;run++))
+	do
+	  for poddatalog in "${POD_CPU_LOGS[@]}"
+		do
+			# Parsing CPU, app metric logs for pod
+			parsePodDataLog ${poddatalog} ${TYPE} ${run} ${ITR}
```

Comment:
Need updates to parsePodDataLog function as it may not work to parse the cpu results as the getmetrics-promql.sh scripts are not collecting the namespace details. 
Same applies for memory logs as well.